app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```
@@ -482,6 +482,13 @@
       "WHATSAPP_WEBHOOK_SUBHEADER": "This token is used to verify the authenticity of the webhook endpoint.",
       "UPDATE_PRE_CHAT_FORM_SETTINGS": "Update Pre Chat Form Settings"
     },
+    "HELP_CENTER": {
+      "LABEL": "Help Center",
+      "PLACEHOLDER": "Select Help Center",
+      "SELECT_PLACEHOLDER": "Select Help Center",
+      "REMOVE": "Remove",
```

Changes based on option 1 of [this suggestion](https://github.com/chatwoot/chatwoot/pull/6903/files#r1172809903)

```suggestion
      "REMOVE": "Remove Help Center ",
```